src/common/components/Tooltip1/Tooltip.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FunctionComponent } from 'react'
```

Comment:
@sanjanadhikari1 why did you use index number in folder name? It doesn't make sense.

Reply:
Because there was Tooltip component already that the other developer were using. It was using the styled component. If I delete it, all others will be broken. That's the reason. But he can use it my tooltip component I think.

Reply:
@sanjanadhikari1 if you need another kind of tooltip, you should customize an existing tooltip component. No need to create a new tooltip component.

Reply:
I don't think we uses two kinds of tooltip in the app. Perhaps you'll have to discuss thing with him as well.

Reply:
Okay let me resolve this issue now.

Reply:
Thanks for your comments :)

Reply:
Fixed and pushed again. You can proceed for checking. Thanks :)